docker/dockerfiles/artifacts/artifact.Dockerfile

```diff
@@ -51,4 +52,11 @@ COPY --from=fe-builder /build/starrocks/output /release/fe_artifacts
 COPY --from=be-builder /build/starrocks/output /release/be_artifacts
 COPY --from=broker-builder /build/starrocks/fs_brokers/apache_hdfs_broker/output /release/broker_artifacts
 
+# download the latest dd-java-agent
+ADD 'https://dtdg.co/latest-java-tracer' /release/fe_artifacts/fe/datadog/dd-java-agent.jar
+
+# Get ddprof for BE profiling
+RUN wget https://github.com/DataDog/ddprof/releases/download/v0.15.3/ddprof-0.15.3-${TARGETARCH}-linux.tar.xz -O ddprof-linux.tar.xz && \
```

Comment:
`curl -s {url} -o - | tar -xf - ddprof/bin/ddprof && mv ...`

also `mv` would fail if `/release/be_artifacts/be/datadog/` not exist.